lib/Steps/Integrations/ReactNative.ts

```diff
@@ -279,9 +279,13 @@ export class ReactNative extends MobileProject {
       null,
       {
         shellPath: '/bin/sh',
-        shellScript:
-          'export SENTRY_PROPERTIES=sentry.properties\\n' +
-          '../node_modules/@sentry/cli/bin/sentry-cli upload-dsym',
+        shellScript:`
+export SENTRY_PROPERTIES=sentry.properties
+INCLUDE_SOURCES_KEY=debug_files.upload.include_sources
```

Comment:
what if we implement that directly on Sentry CLI? does it make sense? Sentry CLI already loads the sentry.properties and uses its configurations, maybe we just need to add support for it in there and it'd work for Android as well?

Reply:
The values from the `properties` file are in the cli, so implementation would be simple if statement.

But I'm not sure if a non-global attribute should be passed thru the `properties`. 

Here it makes sense to me because I'm looking at the properties as settings of the Xcode plugin. 

The android plugin uses the `properties` directly. Also, it uses a different command (react-native gradle), so I'm not sure if it will even need this flag.